File: app/src/main/java/me/zheteng/android/videosaver/parser/ParserUtils.java
package me.zheteng.android.videosaver.parser;

import android.support.annotation.Nullable;

/**
 * 工具类
 */
public class ParserUtils {
    /**
     * Returns true if the string is null or 0-length.
     *
     * @param str the string to be examined
     *
     * @return true if str is null or zero length
     */
    public static boolean isEmpty(@Nullable CharSequence str) {
        if (str == null || str.length() == 0) {
            return true;
        } else {
            return false;
        }
    }

}

File: app/src/main/kotlin/me/zheteng/android/videosaver/parse/ParserUtils.kt
package me.zheteng.android.videosaver.parse

import android.text.TextUtils

/**
 * 工具类
 */
object ParserUtils {
    /**
     * Copied from [TextUtils.isEmpty]
     * Returns true if the string is null or 0-length.

     * @param str the string to be examined
     * @return true if str is null or zero length
     */
    fun isEmpty(str: CharSequence?): Boolean {
        return str == null || str.isEmpty()
    }

}
